recipes/pybob/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pybob" %}
+{% set version = "0.26.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 717cd4e7c2be8ea6a00000a5aac957d8a3bfd77a66a72ce683b2cac225850254
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - descartes
+    - fiona
+    - gdal
+    - geopandas
+    - h5py
+    - matplotlib
```

Comment:
Unless you need a PyQt GUI, better to depend on `matplotlib-base`.

Reply:
Updated!